pkgs/development/tools/build-managers/cmake/check-pc-files.sh

```diff
@@ -0,0 +1,13 @@
+pcfileCheckPhase() {
+    for file in $(find ''${!outputDev} -iname "*.pc" -print); do
+      local grepout=$(grep --line-number '}//' $file || true)
```

Comment:
Can you add a comment explaining this?

Basically we try to find double slashes which normally are never generated?

Reply:
This is more of a heuristic than anything, but the problematic values are usually of the form `${prefix}//nix/store/...`, generated from `${prefix}/@CMAKE_INSTALL_...@` in the source tree, or the same with `exec_prefix`. Looking for just the double slashes doesn’t work because pkg-config files occasionally contain URLs, but `[^:]//` would probably work—or, on the stricter side, `prefix}//nix/store/`.

Reply:
I think we should make this stricter if possible to reduce the chance for false positives.